fe/src/App.js

```diff
@@ -1,24 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import CREATE_DAILY_REPORT from "./pages/CREATE_DAILY_REPORT";
```

Comment:
- change this to Pascal Case
```suggestion
import CreateReport from "./pages/reports/create";
```